server/sonar-db-dao/src/main/java/org/sonar/db/component/ComponentDto.java

```diff
@@ -174,11 +173,33 @@ public ComponentDto setUuidPath(String s) {
     return UUID_PATH_SPLITTER.splitToList(uuidPath);
   }
 
-  @Override
-  public String key() {
+  /**
+   * Used my MyBatis mapper
+   */
+  private String getKee(){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "getKee" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)